scripts/recognize.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+PKG = 'embodied_attention'
+
+import roslib; roslib.load_manifest(PKG)
+import rospy
+
+import sys
+import numpy as np
+
+from sensor_msgs.msg import Image
+from std_msgs.msg import String
+
+from cv_bridge import CvBridge, CvBridgeError
+from skimage.transform import resize
+
+from keras.applications.mobilenet import MobileNet
```

Comment:
so, keras is still used for the recognizer? We should probably go to a full tensorflow solution, as Keras isn't maintained anymore